packages/expo-file-system/src/next/FileSystem.types.ts

```diff
@@ -18,7 +18,7 @@ export declare class Directory {
   constructor(...uris: (URI | File | Directory)[]);
 
   /**
-   * Represents the directory URI.
+   * Represents the directory URI. The field is read-only, but can be updated by calling some methods, like `move`.
```

Comment:
Maybe something like:
```suggestion
   * Represents the directory URI. The field is read-only, but it may change as a result of calling some methods such as `move`.
```